src/Drivers/DriverAbstract.php

```diff
@@ -2,16 +2,35 @@
 
 namespace KolayIK\Auth\Drivers;
 
+use Carbon\Carbon;
+use Exception;
+use KolayIK\Auth\Entity\RefreshToken;
+use KolayIK\Auth\Logger\AuthLogger;
 use KolayIK\Auth\Providers\Storage\StorageInterface;
 
-class DriverAbstract
+/**
+ * Class DriverAbstract
+ * @package KolayIK\Auth\Drivers
+ */
+abstract class DriverAbstract implements DriverInterface
 {
+    /** @var AuthLogger */
+    protected $logger;
     private $config;
     private $cache;
 
     protected $ttl = null;
     protected $refreshTtl = null;
 
+    /**
+     * DriverAbstract constructor.
+     * @param  AuthLogger  $logger
+     */
+    public function __construct(AuthLogger $logger)
```

Comment:
Also please check test, because of this change `DriverAbstractTest` is throwing error and one test is failing